client/blocks/inline-help/popover.jsx

```diff
@@ -188,11 +188,12 @@ class InlineHelpPopover extends Component {
 
 		return (
 			<Popover
-				isVisible
 				onClose={ this.props.onClose }
 				position="top left"
-				context={ this.props.context }
+				expandOnMobile={ false }
+				noArrow={ false }
 				className={ classNames( 'inline-help__popover', popoverClasses ) }
+				getAnchorRect={ () => this.props.context.getBoundingClientRect() }
```

Comment:
would `getAnchorRect={ this.props.context.getBoundingClientRect }` be equivalent?

Reply:
Unfortunately not, that would require binding `getBoundingClientRect`.

Reply:
Cool :)